lhotse/supervision.py

```diff
@@ -24,22 +34,27 @@
 )
 
 
-@dataclass
-class AlignmentItem:
+class AlignmentItem(NamedTuple):
     """
     This class contains an alignment item, for example a word, along with its
     start time (w.r.t. the start of recording) and duration. It can potentially
     be used to store other kinds of alignment items, such as subwords, pdfid's etc.
-
-    We use dataclasses instead of namedtuples (even though they are potentially slower)
-    because of a serialization bug in nested namedtuples and dataclasses in Python 3.7
-    (see this: https://alexdelorenzo.dev/programming/2018/08/09/bug-in-dataclass.html).
-    We can revert to namedtuples if we bump up the Python requirement to 3.8+.
     """
 
     symbol: str
     start: Seconds
     duration: Seconds
+    score: Optional[float] = None
```

Comment:
What does the score indicate?

Reply:
It's aligner-specific "confidence score", in case of torchaudio they suggested to provide the average token probability for a given segment in their tutorial; but it could be also computed differently. I'll add a proper note.